WiseUnpacker/SabreTools.Serialization.Deserializers/WiseSectionHeader.cs

```diff
@@ -87,41 +92,210 @@ public class WiseSectionHeader : BaseBinaryDeserializer<SectionHeader>
             //Seek back to the beginning of the section
             data.Seek(initialOffset, 0);
 
-            header.UnknownValue0 = data.ReadUInt32LittleEndian();
+            header.UnknownDataSize = data.ReadUInt32LittleEndian();
             header.SecondExecutableFileEntryLength = data.ReadUInt32LittleEndian();
             header.UnknownValue2 = data.ReadUInt32LittleEndian();
             header.UnknownValue3 = data.ReadUInt32LittleEndian();
             header.UnknownValue4 = data.ReadUInt32LittleEndian();
             header.FirstExecutableFileEntryLength = data.ReadUInt32LittleEndian();
             header.MsiFileEntryLength = data.ReadUInt32LittleEndian();
-            if (headerLength == 6)
-                return header;
+            // Read common values
 
-            header.UnknownValue7 = data.ReadUInt32LittleEndian();
-            header.UnknownValue8 = data.ReadUInt32LittleEndian();
-            if (headerLength == 8)
-                return header;
-
-            header.UnknownValue9 = data.ReadUInt32LittleEndian();
-            header.UnknownValue10 = data.ReadUInt32LittleEndian();
-            header.UnknownValue11 = data.ReadUInt32LittleEndian();
-            header.UnknownValue12 = data.ReadUInt32LittleEndian();
-            header.UnknownValue13 = data.ReadUInt32LittleEndian();
-            header.UnknownValue14 = data.ReadUInt32LittleEndian();
-            header.UnknownValue15 = data.ReadUInt32LittleEndian();
-            header.UnknownValue16 = data.ReadUInt32LittleEndian();
-            header.UnknownValue17 = data.ReadUInt32LittleEndian();
-            if (headerLength == 17)
-                return header;
+            if (headerLength > 6)
```

Comment:
Looking at the original code, I think you meant to do `>=` for this and the following.